csfieldguide/templates/interactives/sorting-algorithms-configurator.html

```diff
@@ -0,0 +1,52 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+  <div class="row my-3 container-fluid justify-content-center">
+    <div class="row">
+      <div class="col-12">
+        <h4>{% trans 'Sorting Algorithms Configurator' %}</h4>
+      </div>
+
+      <!-- Data type -->
+      <div class="col-12 col-md-6">
+        <div class="form-group">
+          <small id="data-help" class="form-text text-muted">{% trans 'Choose how the data gets shuffled:' %}</small>
+          <select id="data-selector" class="form-control" aria-describedby="data-help">
```

Comment:
That must be a browser setting - it's green for me in both chrome and FF. Since it also won't be unique to this interactive I'm not going to change anything

Reply:
It's not happening on any other interactives for me :woman_shrugging: but if you can't replicate then there's not much you can do I guess..

Reply:
Weird...